### ISSUE TITLE
Arctos IPT resources have lost SQL table linkage

### ISSUE THREAD

**Issue by dbloom** [ArctosDB/arctos#9174]
Hey @dustymc 

It appears that each and every Arctos resource on the VertNet IPT has lost it's connection to your SQL tables. It would appear the most of them are ok in regard to the last publication date. It doesn't appear that anyone has published 0 records, so that is good, but when I attempt to refresh individual resources I receive two errors:

1. The number of columns changed from 91 to 0
2. Failed to analyze the source {}

So it would appear that the IPT can no longer identify individual collections or table, to say nothing of the data in those tables.

Question: this seems to be happening more frequently over the last year. Feels like we do this dance at least once a month now. Last time we solved it when you "kicked the box" for me. Perhaps that is the solution for this time around, but I am concerned that this is happening waaaay too often. Is something happening at TACC? Do we need to update firewall settings? Something else?

**Comment by dustymc** [comment]
You're copied on an email exchange with TACC, they need information (nobody can remember what IP should have the tunnel).

FWIW the report that's supposed to prevent the 'everything failed...' flavor of this ran recently and looks happy:

<img width="1653" height="148" alt="Image" src="https://github.com/user-attachments/assets/19b093ec-8fb2-48a0-924b-54186027083e" />

**Comment by dustymc** [comment]
merge-->https://github.com/ArctosDB/internal/issues/456
